02-Electrophysiology/02-Imported Tables - Interactive.py

```diff
@@ -76,7 +74,7 @@
 key = keys[0]
 key
 
-filename = 'data/data_{mouse_id}_{session_date}.npy'.format(**key)
+filename = '../data/data_{mouse_id}_{session_date}.npy'.format(**key)
```

Comment:
These changes seemed to be out of sync with the Jupyter notebook.  Were you able to sync with `jupytext`?  If `jupytext` is an added burden, we can remove the paired Python scripts.

Reply:
It is an added burden, I'd suggest to remove